django_app/tests/conftest.py

```diff
@@ -74,9 +74,7 @@ def s3_client():
 @pytest.fixture()
 def chat_history(alice: User) -> ChatHistory:
     session_id = uuid.uuid4()
-    chat_history = ChatHistory.objects.create(id=session_id, users=alice)
-    yield chat_history
```

Comment:
In this case, it's really not doing anything - the pytest-django library is tearing down the objects. I'm not 100% on the File because of the side effects, which is why I left it, but I'm sure of this one.